android/build.gradle

```diff
@@ -10,13 +10,19 @@ buildscript {
 
 apply plugin: 'com.android.library'
 
+def _reactNativeVersion = rootProject.ext.reactNative ?: '+';
```

Comment:
Not quite sure what's going on here but this is causing the following error on build now:

> Error:(13, 1) A problem occurred evaluating project ':react-native-linear-gradient'.
> Cannot get property 'reactNative' on extra properties extension as it does not exist

Reply:
Odd... Are you sure it's due to this PR?

Reply:
@designorant and @GantMan I described the issue and how to fix it in #232 
I also sent you a message on Gitter